### PR TITLE
[jsk_pr2_startup] Run email_topic.py and smach_to_mail.py by default

### PR DESCRIPTION
**Pull request by tkmtnt7000**

### CODE REVIEW

File: jsk_pr2_robot/jsk_pr2_startup/pr2.launch [hunk 1]
```diff
@@ -233,4 +233,18 @@
...
 
+  <!-- send email by rostopic -->
+  <node name="email_topic" pkg="jsk_robot_startup" type="email_topic.py" output="screen">
+    <rosparam>
+      email_info: /var/lib/robot/email_topic.yaml
+    </rosparam>
+  </node>
+
+  <!-- send email, twitter with smach -->
+  <node name="server_name" pkg="jsk_robot_startup" type="smach_to_mail.py" output="screen">
```
Comment (knorth55) [inline jsk_pr2_robot/jsk_pr2_startup/pr2.launch]: name should be more detailed like smach_to_mail_server

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: jsk_pr2_robot/jsk_pr2_startup/pr2.launch (hunk 1 above)
Comment (tkmtnt7000) [reply jsk_pr2_robot/jsk_pr2_startup/pr2.launch]: OK. I'll fix it.

Comment (tkmtnt7000) [reply jsk_pr2_robot/jsk_pr2_startup/pr2.launch]: I fixed in https://github.com/knorth55/jsk_robot/pull/338/commits/576a8df7977c7c7c0bf5338bfbf818bbdc5bab15

**Comment by knorth55** [comment]
I rebased and cherry-picked these commits into `pr1040` branch.

**Pull request closed without merging**